rlclientlib/live_model_impl.cc

```diff
@@ -100,6 +100,48 @@ namespace reinforcement_learning {
       status);
   }
 
+  int live_model_impl::request_continuous_action(const char* event_id, const char* context, unsigned int flags, continuous_action_response& response, api_status* status)
+  {
+    if (!_configuration.get_bool(name::CONTINUOUS_ACTIONS_ENABLED, value::DEFAULT_CONTINUOUS_ACTIONS_ENABLED))
```

Comment:
Why do we need explicit flag? We already have model_type() which seems to be answering this question.

Reply:
It's a safeguard to make sure continuous actions is not being used until changes are implemented in the joiner/trainer also

Reply:
But also allows us to turn it on for testing etc, where as if we just use the model type we can't enable/disable continuous actions usage.

Reply:
We don't need such safeguard, APS will work fine if we send CA events to it.
This extra required knob will lead to fragility down the line.

Reply:
Good to know, removed